qtconsole/frontend_widget.py

```diff
@@ -482,10 +482,8 @@ def _handle_execute_reply(self, msg):
             status = content['status']
             if status == 'ok':
                 self._process_execute_ok(msg)
-            elif status == 'error':
-                self._process_execute_error(msg)
-            elif status == 'aborted':
```

Comment:
I think it probably should. Displaying tracebacks should only be done when `error` messages arrive.

While calling `_process_execute_aborted` makes sense to be called if it does something other than display the traceback, displaying the traceback for user code isn't part of that.

I think it's a bug that KeyboardInterrupt sets status=aborted, anyway. I'm not sure how that got in there.